doc/doxygen/install/install-linux.doxygen

```diff
@@ -203,6 +206,14 @@
   mkdir OpenMS-build
   cd OpenMS-build
   \endcode
+  
+  Install the qt4-webkit, qt4-dev-tools and libqtwebkit-dev
+
+   \code
+   sudo apt-get install qt4-webkit
+   sudo apt-get qt4-deb-tools
+   sudo apt-get libqtwebkit-dev
+   \endcode
```

Comment:
I totally agree with @hroest .
This is why we introduced two sections for Ubuntu/Debian (> 10.04 <= 15)	and Ubuntu/Debian (>= 16.04 <= 16.10)	
 :-1: